server_manager/install_scripts/build.action.sh

```diff
@@ -16,7 +16,7 @@
 
 set -eu
 
-readonly OUT_DIR="${BUILD_DIR}/server_manager/www/sh/"
+readonly OUT_DIR="${BUILD_DIR}/server_manager/install_scripts/"
```

Comment:
Shall we call it `server_install_scripts` since they are installing a server instead of the manager?

Reply:
Better leave that to another PR, to avoid too many changes that can break things.